account_invoice_partner_no_autofollow/tests/test_account_move_no_autofollow.py

```diff
@@ -70,7 +70,7 @@ def test_in_invoice_create(self):
         )
 
         self.assertEqual(
-            len(move.message_follower_ids.ids), 0, msg="Count followers must be equal 0"
+            len(move.message_follower_ids.ids), 1, msg="Count followers must be equal 0"
```

Comment:
Fix spelling: Follower count must be equal **to** 0
Here and in all other message strings.
You are comparing to 1 but message says "must be equal **to** 0"
And why 0? Using fixed numbers here might play a bad trick on you.
What if some other module will add another partners as followers? Eg "accountant assistant" etc?

Reply:
fix